lib/granite/form/model/attributes/represents.rb

```diff
@@ -3,15 +3,6 @@ module Form
     module Model
       module Attributes
         class Represents < Attribute
-          TYPES = {
```

Comment:
Most of this is moved to `lib/granite/form/model/attributes/reflections/represents/build_type_definition.rb` with very slight modifications.